toptal/picasso
# style(Table): update color of selected row

[FX-2060]
[Abstract](https://app.abstract.com/share/60c30375-c8cb-4d07-b9c9-444dad37d477?mode=design&sha=0aeb05e01da837c228249c74cc79ec21280f8b6d)

### Description

- Updating the background color of the selected row in the table according to the BASE 2.0 design.
- Overwrite default MUI hover color of selected row
- moved visual tests to happo
### How to test

- check storybook and compare it with design. Try remove `hover` from `<Row>` at selectable table example.

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![image](https://user-images.githubusercontent.com/6830426/133999430-7ff2676e-0a7f-45e9-9ad3-1981476bc3d4.png) | ![image](https://user-images.githubusercontent.com/6830426/133999497-6598c2d5-db26-4e85-8763-7ea9c35c60e6.png) |

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-2060]: https://toptal-core.atlassian.net/browse/FX-2060
